### PR TITLE
[RAPTOR-3132] custom.py files can be found in nested directories

### PR DESCRIPTION
**Pull request by 4dahalibut**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

Sometimes users like messing up in this way, so instead of failing, we'll let them mess up and adapt.

### CODE REVIEW

File: tests/drum/test_custom_model.py [hunk 1]
```diff
@@ -480,30 +482,32 @@ def wait_for_server(url, timeout, process_holder):
...
+            (XGB, BINARY, PYTHON, None, False),
+            (PYTORCH, REGRESSION, PYTHON, None, False),
+            (PYTORCH, BINARY, PYTHON, None, False),
+            (RDS, REGRESSION, R, None, True),
+            (RDS, BINARY, R, None, False),
+            (CODEGEN, REGRESSION, NO_CUSTOM, None, False),
+            (CODEGEN, BINARY, NO_CUSTOM, None, False),
+            (MULTI_ARTIFACT, REGRESSION, PYTHON_LOAD_MODEL, None, False),
+            (PYPMML, REGRESSION, NO_CUSTOM, None, True),
+            (PYPMML, BINARY, NO_CUSTOM, None, False),
```
Comment (meltzg) [inline tests/drum/test_custom_model.py:501]: what's the logic for what is False and what is True?

Comment (4dahalibut) [reply tests/drum/test_custom_model.py:501]: it should work for the true ones. I'm kinda just letting it spin, but I don't think nested dirs is gon work for java and R for now...

**Comment by 4dahalibut** [comment]
@yakov-g I just wanted to toss that in here so that people could run inference against vizai models without having to muck with pandas.

**Comment by yakov-g** [comment]
> Is dataset change intentional?

> @yakov-g I just wanted to toss that in here so that people could run inference against vizai models without having to muck with pandas.

But what is the problem? I can run inference

File: tests/drum/test_custom_model.py (hunk 1 above)
Comment (yakov-g) [reply tests/drum/test_custom_model.py:501]: > what's the logic for what is False and what is True?

+1 : create a separate usecase, THen you won't need to put these dummy False.

And then, ode day, we could add R.

**Comment by 4dahalibut** [comment]
I didn't implement the feature for R.

**Comment by 4dahalibut** [comment]
We just don't have R custom training models yet.

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**2.49** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**